src/App.jsx

```diff
@@ -1,35 +1,34 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
+import './App.css';
+import FetchingData from '../../Burns-Week-4-HW/src/components/FetchingData';
+import PostingData from '../../Burns-Week-4-HW/src/components/PostingData';
```

Comment:
These imports were breaking the app. I noticed it was a quick fix for me so I went ahead and did it. Keep in mind that I will ask you to resubmit and I won't look at the assignment until it runs.

```suggestion
import FetchingData from './components/FetchingData';
import PostingData from './components/PostingData';
```